utils/src/print.c

```diff
@@ -1006,6 +1006,12 @@ static int print_volume(int fd)
 
 	print_super_block(super, SCOUTFS_SUPER_BLKNO);
 
+	if (!(le64_to_cpu(super->flags) && SCOUTFS_FLAG_IS_META_BDEV)) {
+	    fprintf(stderr,
```

Comment:
full tabs for indentation, not spaces, and let's set ret for an error so it's not 0 from before.. maybe -EINVAL?